scripts/src/tests/services/users.spec.ts

```diff
@@ -121,14 +121,14 @@ describe("api tests for v2 users", async () => {
 
 	test("updateUser", async () => {
 		const testApp = await helpers.createApp(generateId(IdPrefix.App));
-		const newWalletAddress = "new_address_must_be_56_characters____bla___bla___bla____";
+		const newWalletAddress = generateRandomString({ length: 56 });
```

Comment:
@nitzantomer happy? (That was a complete waste of my time. thank you). Now please approve and merge it if you're already at it.

Reply:
@liorama, no one told you to add functionality which isn't needed. you could have just left it with the old value which had 56 chars...

Reply:
@nitzantomer No, that would have failed the new test, it needed to be a new address every time.